cpg_workflows/jobs/align.py

```diff
@@ -346,7 +346,12 @@ def _align_one(
     job_name = f'{job_name} {alignment_input}'
     j = b.new_job(job_name, job_attrs)
 
-    nthreads = STANDARD.set_resources(
+    if get_config()['resource_overrides']['align_use_highmem']:
```

Comment:
Should probably use `.get('align_use_highmem')` here, so the tests can get away without a dummy entry.